src/AppContainer.tsx

```diff
@@ -44,6 +49,12 @@ const AppContainer = (): React.ReactElement => {
           ))}
           {NETWORK === "ropsten" && (
             <FeatureFlag name="MAGIC_DEMO">
+              <Route path="/demo/viewer" exact>
+                {isMagicDemoDocumentVerified ? <DemoViewerPage /> : <Redirect to="/demo/verify" />}
+              </Route>
```

Comment:
dont do this here, do it in the sagas. refer to certificate.js in the sagas for reference.

Reply:
that comment might not be valid.

Reply:
not relevant anymore, will remove